FWCore/Framework/src/EDConsumerBase.cc

```diff
@@ -268,7 +270,7 @@ ProductResolverIndexAndSkipBit EDConsumerBase::indexFromIfExactMatch(EDGetToken
       return info.m_index;
     }
   }
-  return ProductResolverIndexAndSkipBit(edm::ProductResolverIndexInvalid, false);
+  return ProductResolverIndexAndSkipBit(edm::ProductResolverIndexInitializing, false);
```

Comment:
The reason for this change is not immediately clear to me.

Reply:
It is now possible for info.m_index to return `ProductResolverIndexInvalid` (e.g. if told to `@skipCurrentProcess` and that was determined to match no data product). As such, I needed a different denotation that there is a problem where the token was somehow not properly initialized.

Reply:
Ok